cert/CACreationScript.sh

```diff
@@ -108,25 +108,33 @@ openssl ca -create_serial -cert $PI.crt -keyfile $PI.pkey -outdir $CSR -batch -o
 #PAL: TODO
 
 #Device Intermediate
-CBSD=wif_cbsd_ca
+CBSD=cbsd_ca
 CBSD_SIGN=$CBSD$SIG
 
-openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Device CA-1" -config openssl.cnf
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA CBSD CA-1" -config openssl.cnf
 
 openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batch -out $CBSD.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_SIGN -config openssl.cnf -in $CBSD.csr
 
 #Device EE
-CBSD_EE=wif_cbsd_req
+CBSD_EE=cbsd_req
 CBSD_EE_SIGN=$CBSD_EE$SIG
 
-openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Device End-Entity Example" -config openssl.cnf
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=CBSD End-Entity Example" -config openssl.cnf
 
 openssl ca -create_serial -cert $CBSD.crt -keyfile $CBSD.pkey -outdir $CSR -batch -out $CBSD_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_EE_SIGN -config openssl.cnf -in $CBSD_EE.csr
 
-#TODO: manufacturer intermediate signed by $CBSD, and EE cert signed by manufacturer intermediate
-
+#Device OEM Intermediate
+CBSD_OEM=cbsd_oem_ca
+CBSD_OEM_SIGN=$CBSD_OEM$SIG
 
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD_OEM -out $CBSD_OEM.csr -keyout $CBSD_OEM.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA CBSD OEM CA-1" -config openssl.cnf
 
+openssl ca -create_serial -cert $CBSD.crt -keyfile $CBSD.pkey -outdir $CSR -batch -out $CBSD_OEM.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_OEM_SIGN -config openssl.cnf -in $CBSD_OEM.csr
```

Comment:
This is a CA issued by the CBSD Intermediate, so it's being signed by the CBSD crt/pkey files (the only CA cert _not_ issued directly by the Root).